From akpm@linux-foundation.org  Tue Jun  2 12:02:51 2009
From: akpm@linux-foundation.org
Date: Tue, 12 May 2009 15:27:59 -0700
Subject: USB: sierra: fix printk warning
Subject: + drivers-usb-serial-sierrac-fix-printk-warning.patch added to -mm tree
To: mm-commits@vger.kernel.org
Cc: akpm@linux-foundation.org, greg@kroah.com
Message-ID: <200905122247.n4CMlFo9011327@imap1.linux-foundation.org>


From: Andrew Morton <akpm@linux-foundation.org>

drivers/usb/serial/sierra.c: In function 'sierra_write':
drivers/usb/serial/sierra.c:375: warning: format '%d' expects type 'int', but argument 5 has type 'size_t'

Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Cc: Elina Pasheva <epasheva@sierrawireless.com>
Cc: Rory Filer <rfiler@SierraWireless.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/sierra.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/serial/sierra.c
+++ b/drivers/usb/serial/sierra.c
@@ -415,7 +415,7 @@ static int sierra_write(struct tty_struc
 
 	portdata = usb_get_serial_port_data(port);
 
-	dev_dbg(&port->dev, "%s: write (%d bytes)\n", __func__, writesize);
+	dev_dbg(&port->dev, "%s: write (%zd bytes)\n", __func__, writesize);
 
 	spin_lock_irqsave(&portdata->lock, flags);
 	dev_dbg(&port->dev, "%s - outstanding_urbs: %d\n", __func__,
